ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,72 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the Antrea project.
+ It deploys Kubernetes with the Antrea image and tests whether the
+ OVS has hardware offload capabilities or not.
```

Comment:
I don't understand the extra white spaces at the beginning of lines throughout this file?

Reply:
sorry, my bad. removed them.